pina/label_parameter.py

```diff
@@ -0,0 +1,197 @@
+""" Module for LabelTensor """
+from typing import Any
+import torch
+from torch import Tensor
+
+
+class LabelParameter(torch.nn.Parameter):
```

Comment:
I am not convinced at all about this class. It seems a copy of LabelTensor. Is it really needed?